TheWarWithin/DemonHunterVengeance.lua

```diff
@@ -1452,24 +1495,28 @@ spec:RegisterAbilities( {
             gain( health.max * 0.4, "health" )
 
             if talent.demonsurge.enabled then
-                applyBuff( "demonsurge_soul_cleave", buff.metamorphosis.remains )
-                applyBuff( "demonsurge_spirit_bomb", buff.metamorphosis.remains )
-            end
+                local metaRemains = buff.metamorphosis.remains
 
-            if talent.demonic_intensity.enabled then
-                applyBuff( "demonsurge_consuming_fire", buff.metamorphosis.remains )
-                applyBuff( "demonsurge_fel_desolation", buff.metamorphosis.remains )
-                applyBuff( "demonsurge_sigil_of_doom", buff.metamorphosis.remains )
-            end
+                applyBuff( "demonsurge_soul_cleave", metaRemains )
+                applyBuff( "demonsurge_spirit_bomb", metaRemains )
```

Comment:
The buff names here are incorrect and should be `demonsurge_soul_sunder` and `demonsurge_spirit_burst`.